mkosi/backend.py

```diff
@@ -269,6 +280,10 @@ class PartitionTable:
 
     grain: int = 4096
 
+    def __post_init__(self) -> None:
+        if not sfdisk_grain_is_supported():
+            self.grain = 1048576 # Use sfdisk default grain size of 1 MiB.
```

Comment:
Maybe a matter of preference in terms of readability
```suggestion
            self.grain = 1024 ** 2 # Use sfdisk default grain size of 1 MiB.
```
I think Python would constant fold this.